### ISSUE TITLE
Brief and guidelines for video production

### ISSUE THREAD

**Issue by natalieworth** [MozillaFoundation/Design#757]
### Intro

We need a brief and guidelines for video production for the internal and external process. Some initial things to consider: working with contractors and file type handover, open/ closed captions and typography for them, are the videos being localized and in what languages, where will the videos be displayed and who will upload them (Instagram, Youtube, etc.), bumpers, lower-thirds, etc.

cc @audreymofo

**Comment by kristinashu** [comment]
This should include your guidelines about localisation https://docs.google.com/document/d/1z_0EPR2w5CxTeiXnaasf3jR0pMnoZCRHdD-yHxEWjgc/edit

**Comment by kristinashu** [comment]
Here are Sabrina's collaboration guidelines too https://foundation.mozilla.org/en/docs/brand/process/collaboration/

**Comment by natalieworth** [comment]
Also adding to combine this in here from #741: Add a video/ motion/ gif specific page with on Wagtail for agency/ contractor/ marketing purposes.

**Comment by natalieworth** [comment]
I'm creating a rough [document here](https://docs.google.com/document/d/180_I7cj-2C06AwMs4Ullpavi_LeD0prvs6_PoK2Qq7E/edit?usp=sharing) to organize the structure and assets before adding it to wagtail.

**Comment by natalieworth** [comment]
Just awaiting feedback/ review from Audrey once she gets times outside of MozFest. :)

**Comment by natalieworth** [comment]
Audrey had 1 comment to possibly add the animated backgrounds once Eugene is done the first video. I can make those downloadable once I get final files from him.

I also spoke with a MoCo designer who is working on another Explains series with the MoCo team and we were thinking for alignment, to use ITC Franklin Gothic instead of Acumin, so I've updated that too.

**Comment by natalieworth** [comment]
I've updated based on MoCo and Audrey's comments and guidelines are in here https://foundation.mozilla.org/en/docs/brand/process/video-production-guidelines/

@sabrinang or @kristinashu If you have time, could you R+ and I will close :D Noting too that I can revise this if needed after the first video is done/ if there's additional feedback.

**Comment by sabrinang** [comment]
I reviewed the link and content looks good. I like the how-to google doc linked in the process section too. 

Some feedback:
- in typography, ITC Franklin Gothic isn't open source so should we link how they access it or which copy do we use?
- in deliverables file hand-off section, if there is an example of what "organized" to us looks like _(naming, folder etc. that is desired)_ that could be helpful in preventing a misinterpreted standard of organization

**Comment by natalieworth** [comment]
Thanks Sabs! I updated the section saying that ITC Franklin Gothic will be provided via Mozilla and then a link to Nunito Sans (on Google Fonts). 

For the deliverables section, it mayyy depend on what the project is and who we're working with so I clarified with: "Depending on the project, the desired hand-off will be coordinated with the producer and a designer. Files can be uploaded to the Mozilla Foundation site via a link provided by a Mozilla designer."

In the past, I've just made a link for Tyler to upload his files and then reordered them in our private folders.

Going to close this with those last updates (but we can also revisit if we need to update in the future). :)
